packages/blade/docs/tokens/CssVariables.stories.mdx

```diff
@@ -0,0 +1,813 @@
+import { Meta, DocsContainer } from '@storybook/addon-docs';
+import { useTheme, BladeProvider } from '../../src//components';
+import { paymentTheme, bankingTheme } from '../../src//tokens';
+
+<Meta
+  title="Tokens/CSS Variables"
+  parameters={{
+    docs: {
+      container: ({ children, context }) => {
+        const getThemeTokens = () => {
+          if (context.globals.themeTokenName === 'paymentTheme') {
+            return paymentTheme;
+          }
+          if (context.globals.themeTokenName === 'bankingTheme') {
+            return bankingTheme;
+          }
+          return null;
+        };
+        return (
+          <DocsContainer context={context}>
+            <BladeProvider
+              key={`${context.globals.themeTokenName}-${context.globals.colorScheme}`}
+              themeTokens={getThemeTokens()}
+              colorScheme={context.globals.colorScheme}
+            >
+              {children}
+            </BladeProvider>
+          </DocsContainer>
+        );
+      },
+    },
+  }}
+/>
+
+# ✨ CSS Variables
+
+All of the Blade tokens are also available as CSS Variables and can be used by projects that do not want to/cannot handle styling with JS
+
+## Available Themes
+
+- Blade supports 2 themes: **Payment** and **Banking**
+- Each of these 2 themes have a `Light` and a `Dark` mode
+- Depending on screen breakpoints, you can also have different tokens for `Mobile` and `Desktop`
+
+Considering these permutations, we have exposed the following CSS files which you can directly import into your projects:
+
+- **paymentThemeLightDesktop**
+- **paymentThemeLightMobile**
+- **paymentThemeDarkDesktop**
+- **paymentThemeDarkMobile**
+- **bankingThemeLightDesktop**
+- **bankingThemeLightMobile**
+- **bankingThemeDarkDesktop**
+- **bankingThemeDarkMobile**
+
+## Usage
+
+1. Import the CSS file inside your project
+
+```jsx
+import '@razorpay/blade/css/paymentThemeLightDesktop.css';
+```
+
+2. Use the tokens in your css
+
+```css
+.text {
+  color: var(--colors-action-text-primary-default);
+  padding: var(--spacing-1);
+  font-size: var(--typography-fonts-size-75);
+}
+```
+
+> **Note:** Since we depend on CSS variables for our tokens, please ensure that you check for the minimum supported browser versions for CSS variables and handle that approriately.
+> Refer: [caniuse.com/css-variables](https://caniuse.com/css-variables)
+
+## Example
+
+This is an example CSS file for `paymentThemeLightDesktop.css`
+
+> **Note**: This is not the actual `paymentThemeLightDesktop.css` but just a dummy version of it. The actual values of these variables can change over time as we evolve our themes & tokens.
```

Comment:
My recommendation here is to avoid including the actual values below, makes the doc too long (and also harder to maintain) and people interested in knowing the actual variables and values would rather look at the actual file. I think few properties as an example should be fine to give a general idea.

Reply:
Makes sense. Done!